.eslintrc.js

```diff
@@ -1,89 +1,14 @@
-module.exports = {
-  'env': {
-    'es6': true,
-    'node': true
-  },
-  'parser': 'babel-eslint',
-  'extends': [
-    'eslint:recommended',
-    'plugin:flowtype/recommended',
-    'plugin:react/recommended',
-    'plugin:react-native/all'
-  ],
-  'parserOptions': {
-    'ecmaFeatures': {
-      'experimentalObjectRestSpread': true,
-      'jsx': true
-    },
-    'sourceType': 'module'
-  },
-  'plugins': [
-    'react',
-    'react-native',
```

Comment:
If we re-enable the react rules, we'll need to re-enable the react and react-native plugins as well.

Reply:
They have been added back now with the latest commit